src/node/catchup_test.ml

```diff
@@ -137,9 +137,11 @@ let teardown () =
   Logger.debug_ "end of teardown"
 
 let _tic filler_function n name verify_store =
-  Tlogcommon.tlogEntriesPerFile := 101;
+  let open Tlog_map in
+  let node_id = "node_name" in
+  TlogMap.make ~tlog_max_entries:101 _dir_name _tlx_dir node_id ~check_marker:true >>= fun tlog_map ->
```

Comment:
Did you add the construction of the tlogmap here so you could base some asserts on it?